docs/6-infrastructure-configuration-management/6.1.2-terraform-backends.md

```diff
@@ -0,0 +1,67 @@
+# Terraform State & Backends
+
+## Sharing workspaces
+
+In the last tutorial from the previous section you learned how to use Terraform Cloud to store a Terraform state remotely. Using remote states makes it possible for teams to share and create automation for Terraform workspaces. Work with your teammates to in the following exercises to get a better understand of how to share a Terraform workspace.
+
+1. Create a GitHub repository for your Terraform and push up your code. You should only include your `.tf` and `.terraform.lock.hcl`.
+
+2. Working with a teammate have them clone your repo, make a change to your Terraform and apply it and push the change up to your repo.
+
+?> Watch the state of the workspace in the Terraform Cloud as they apply the change
+
+3. Run `terraform plan` with your original configuration. Now pull down your teammates change and run `terraform plan` again and observe the difference.
```

Comment:
"Now pull down your teammates change and run `terraform plan` again then observe the difference."